students/21th/hyunchanpark/login.js

```diff
@@ -0,0 +1,42 @@
+const loginButton = document.querySelector(".login-button");
+const inputEls = document.querySelectorAll("input");
+const inputId = document.querySelector("input[type=text]");
+const inputPassword = document.querySelector("input[type=password]");
+
+const emailRegExp =
+  /[a-zA-Z0-9.-_+!]+@[a-zA-Z0-9]+\.[a-zA-Z0-9]{2,}(?:.[a-zA-Z0-9]{2,3})?/;
+
+const passwordRegExp = /[a-zA-Z0-9]{5,100}/;
+
+const isValidateEmailAndPassword = (email, password) => {
+  let isValidEmail = false;
+  let isValidPassword = false;
+
+  if (email.match(emailRegExp)) isValidEmail = true;
+  if (password.match(passwordRegExp)) isValidPassword = true;
+
+  return isValidEmail && isValidPassword;
+};
+
+const buttonCondition = () => {
+  if (isValidateEmailAndPassword(inputId.value, inputPassword.value)) {
+    return true;
+  } else return false;
+};
+
+const opacityAndDisableButton = (value, boolean) => {
+  loginButton.style.opacity = value;
+  loginButton.disabled = boolean;
+};
+
+const handleInputChange = (event) => {
+  event.preventDefault();
+
+  buttonCondition()
+    ? opacityAndDisableButton(1, false)
+    : opacityAndDisableButton(0.3, true);
```

Comment:
- 삼항연산자 활용 잘 해주셨네요! 👍 
```suggestion
  const idValue = inputId.value;
  const passwordValue = inputId.value;
  isValidateEmailAndPassword(idValue, passwordValue)
    ? opacityAndDisableButton(1, false)
    : opacityAndDisableButton(0.3, true);
```